app/views/layouts/nude.html.erb

```diff
@@ -0,0 +1,21 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title><%= page_title %></title>
+    <meta name="viewport" content="width=device-width, initial-scale=1" />
+    <%# Env variables being made accessible to JavaScript %>
+    <%= tag :meta, name: :sentry_dsn, content: ENV["SENTRY_DSN"] %>
+    <%= tag :meta, name: :release_tag,
+        content: "#{ENV.fetch("APP-BASE", "convene")}@#{ENV.fetch("HEROKU_RELEASE_VERSION", "??")}" %>
+
+    <%= csrf_meta_tags %>
+    <%= csp_meta_tag %>
+    <%= javascript_include_tag "application", "data-turbo-track": "reload", defer: true %>
+    <%= stylesheet_link_tag "application", "data-turbo-track": "reload" %>
+    <%= render "layouts/hotjar" %>
+    <meta name="turbo-visit-control" content="reload">
+  </head>
```

Comment:
Might be worth extracting `<head>...</head>` into a partial, and reusing it across layouts.

Reply:
Ooo good point.  We'll need to maybe add in a way to pass in extra nonsense but otherwise it should be good.